### DISCUSSION TITLE
Vertical slider

### DISCUSSION THREAD

**Discussion by benfredwells** [material-components/material-web#5327] [Q&A]
Is there a way to make the md-slider be vertical? Sorry if I've missed something obvious

**Comment by asyncliz** [comment]
Not "officially" supported, but last time I tried I was able to transform rotate it be 90 degrees and I think it all worked.

Your mileage may vary though

**Comment by benfredwells** [comment]
Thanks, that works, kind of - now I have to learn how to have rotated elements get positioned "correctly", which is surprisingly non-obvious. Changing the writing mode handles positioning a little better but lots of other things get a bit borked.

Another problem is that the tooltips are also rotated which feels a bit odd.
